src/SFML/Window/DRM/DRMContext.cpp

```diff
@@ -130,6 +128,308 @@ namespace
         initialized = false;
     }
 
+    void drmFbDestroyCallback(gbm_bo* bo, void* data)
+    {
+        int drmFd = gbm_device_get_fd(gbm_bo_get_device(bo));
+        sf::priv::DrmFb* fb = reinterpret_cast<sf::priv::DrmFb*>(data);
+
+        if (fb->fbId)
+            drmModeRmFB(drmFd, fb->fbId);
+
+        std::free(fb);
```

Comment:
Can we get around manually calling `free` when we don't have access to smart pointers? I couldn't think of a way to get rid of this easily.

Reply:
It's probably OK as long as the semantics are easy to follow. Workarounds are likely more complex, and this would also allow relatively easy migration to `std::unique_ptr` in SFML 3 (especially when using `new/delete`).